portality/api/v2/data_objects/journal.py

```diff
@@ -0,0 +1,201 @@
+from portality import models
+
+from portality.api.v2.data_objects.common_journal_application import OutgoingCommonJournalApplication
+
+# we only have outgoing journals for the moment
+JOURNAL_STRUCT = {
```

Comment:
There's a bit of repetition here, as the structs for different model objects are declared independently.  Structs can overlay, so we should take this opportunity to normalise them in the v2 of the API.

See an example here: https://github.com/DOAJ/doaj/blob/feature/af_redesign__datamodel/portality/models/v2/journal.py#L404 (and follow your nose into the struct declarations)

Reply:
Yep, that's the plan. It was just easier to write them down first in a full version.